core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7566JavaPrerequisiteTest.java

```diff
@@ -0,0 +1,84 @@
+package org.apache.maven.it;
+
+import java.io.File;
+
+import org.apache.maven.shared.verifier.Verifier;
+import org.apache.maven.shared.verifier.util.ResourceExtractor;
+import org.junit.jupiter.api.Test;
+
+/**
+ * This is a test set for <a href="https://issues.apache.org/jira/browse/MNG-7566">MNG-7566</a>.
+ * Similar to {@link MavenITmng4840MavenPrerequisiteTest}.
+ *
+ */
+class MavenITmng7566JavaPrerequisiteTest
+    extends AbstractMavenIntegrationTestCase
+{
+
+    public MavenITmng7566JavaPrerequisiteTest()
+    {
+        super( "[4.0.0-alpha-3,)" );
```

Comment:
which range to take now requiring current Maven master?

Reply:
Makes sense..

Reply:
Next version should be `4.0.0-alpha-3`, but how are snapshots taken into account ?

Reply:
Certain patterns are stripped before comparing with the given version range: https://github.com/apache/maven-integration-testing/blob/bd7ba0adf5f847b394330cfc022ea833e022ebe5/core-it-support/maven-it-helper/src/main/java/org/apache/maven/it/AbstractMavenIntegrationTestCase.java#L189

SNAPSHOT is one of them.